src/lib.rs

```diff
@@ -589,10 +591,10 @@ pub fn sort_build_outputs_topologically(
         .iter()
         .map(|idx| &outputs[idx.index()])
         .for_each(|output| {
-            tracing::info!(
-                "Ordered output: {:?}",
-                output.outputs[0].name().as_normalized()
-            );
+            // tracing::info!(
```

Comment:
will be uncommented when https://github.com/prefix-dev/rattler-build/pull/728
is merged